distributions/otelcol-contrib/manifest.yaml

```diff
@@ -13,6 +13,7 @@ extensions:
     gomod: go.opentelemetry.io/collector v0.42.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/asapauthextension v0.42.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/awsproxy v0.42.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/basicauthextension v0.42.0
```

Comment:
It doesn't exist at this version though, does it?

Reply:
Should I temporarily use commit ref like `v0.0.0-20220119171801-207f2c3db0f4` or what is the recommended approach for specifying unreleased components?

Reply:
I think this would be the first case where a component is added to the same version. I think I would prefer to update all versions to something like what you listed (but with v0.42.0-... instead of v0.0.0-...). If you could do it in two commits, it would be great: the first commit updating all existing deps, the second adding the new component.

@codeboten, this should be included for 0.43.0.

Reply:
For [pseudo-versions](https://go.dev/ref/mod#pseudo-versions) I can either use `v0.0.0` or `v0.42.1-0`. It fails for me if I try to specify `v0.42.0` like `v0.42.0-20220107162206-6a1c2471092b` for contrib deps. 

I can rename versions to `v0.42.1-0.20220119200947-29b515d5d9c0` format if you actually want me to bump them to the latest revision in `main`.

Reply:
The idea is to be able to replace them all at once. Otherwise, one could miss that one of the modules is at a different version, when they all should have the same by the time it's released. If the only one that works for everything is 0.0.0, then so be it :-)

Reply:
Gotcha.